sql/fxa_users_daily_v1.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TEMP FUNCTION udf_mode_last(x ANY TYPE) AS ((
+  SELECT
+    val
+  FROM (
+    SELECT
+      val,
+      COUNT(val) AS n,
+      MAX(offset) AS max_offset
+    FROM
+      UNNEST(x) AS val
+    WITH OFFSET AS offset
+    GROUP BY
+      val
+    ORDER BY
+      n DESC,
+      max_offset DESC
+  )
+  LIMIT 1
+));
+
+WITH
+  windowed AS (
+  SELECT
+    event_date AS submission_date,
+    CURRENT_DATETIME() AS generated_time,
+    user_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    udf_mode_last(ARRAY_AGG(country) OVER w1) AS country
+  FROM
+    -- We've done a one-time import of historical FxA data into static.fxa_amplitude_export_event_date;
+    -- this table contains events exported from Amplitude and ends on 2019-03-17;
+    -- we only consider this table for now, but we will soon be unioning this with
+    -- data from a separate BigQuery project that does daily imports of FxA logs.
+    static.fxa_amplitude_export_event_date
+  WHERE
+    user_id IS NOT NULL
+    AND event_type NOT IN (
+      'fxa_email - bounced', 'fxa_email - click', 'fxa_email - sent',
+      'fxa_reg - password_blocked','fxa_reg - password_common', 'fxa_reg - password_enrolled',
+      'fxa_reg - password_missing','fxa_sms  - sent', 'mktg - email_click',
```

Comment:
Is the double space in `fxa_sms  - sent` a typo?

Reply:
Indeed this is a bug.